sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -138,16 +142,55 @@ export class CASService {
         ),
         SubCategory: "Individual",
         Sin: supplierData.sin,
-        SupplierAddress: [
-          {
-            AddressLine1: formatAddress(supplierData.supplierSite.addressLine1),
-            City: formatCity(supplierData.supplierSite.city),
-            Province: supplierData.supplierSite.provinceCode,
-            Country: "CA",
-            PostalCode: formatPostalCode(supplierData.supplierSite.postalCode),
-            EmailAddress: supplierData.emailAddress,
-          },
-        ],
+        SupplierAddress: [this.getSupplierAddress(supplierData)],
```

Comment:
Minor and just a suggestion. supplier address can be assigned to a constant before using here. 

```ts
const config = await this.getAuthConfig();
      const supplierAddress = this.getSupplierAddress(supplierData);
      const submittedData: CreateExistingSupplierAndSiteSubmittedData = {
        SupplierNumber: supplierData.supplierNumber,
        SupplierAddress: [supplierAddress],
      };
```